tutor/src/screens/teacher-dashboard/plan-details.jsx

```diff
@@ -245,6 +271,27 @@ class CoursePlanDetails extends React.Component {
     );
   }
 
+  // Render the assigned sections.
+  // Show only the first two with a button to show the rest of the sections.
+  renderAssignedSections() {
+    const { plan: { assignedSections } } = this.props;
+
+    if(!assignedSections) return null;
+    return (
+      <SectionsAssignedWrapper>
+        <h6>Sections Assigned</h6>
+        {assignedSections.map((section, i) => 
+          <ul key={i}>
```

Comment:
not super important but its recommended to not use an iterator as `key` unless there's nothing better available.  In this case we could use `section.pathId`